flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -3,22 +3,25 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 52;
 	objects = {
 
 /* Begin PBXBuildFile section */
 		00168AD425E36DDA00BC9D7D /* CoreML.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 00168AD325E36DDA00BC9D7D /* CoreML.framework */; };
 		00168AD625E36DF700BC9D7D /* libc++.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = 00168AD525E36DEC00BC9D7D /* libc++.tbd */; };
-		00168AD825E3752300BC9D7D /* ios_backend_fw_static in Frameworks */ = {isa = PBXBuildFile; fileRef = 00168AD725E374ED00BC9D7D /* ios_backend_fw_static */; };
-		035547E55D1857EA2B3C8779 /* Pods_RunnerTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 10CDB3E2A91DF927CD274CD3 /* Pods_RunnerTests.framework */; };
 		1498D2341E8E89220040F4C2 /* GeneratedPluginRegistrant.m in Sources */ = {isa = PBXBuildFile; fileRef = 1498D2331E8E89220040F4C2 /* GeneratedPluginRegistrant.m */; };
+		19C134532876922700263A48 /* backend_bridge_fw.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = 19C1344F2876919E00263A48 /* backend_bridge_fw.xcframework */; };
+		19C134542876922700263A48 /* backend_bridge_fw.xcframework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 19C1344F2876919E00263A48 /* backend_bridge_fw.xcframework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		19C1345E2878151A00263A48 /* libtflitebackend.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = 19C1345D2878151A00263A48 /* libtflitebackend.xcframework */; };
+		19C1345F2878151A00263A48 /* libtflitebackend.xcframework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 19C1345D2878151A00263A48 /* libtflitebackend.xcframework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
Xcode links all of the bundled libraries automatically.
I tried adding second instance of TFLite backend to the app. Xcode emitted a warning about duplicated symbols but there were no errors.
I think it shouldn't be an issue because we load libraries and its exported functions manually, and stuff inside libraries should already be linked to itself.